src/apply_manual_codes.py

```diff
@@ -222,15 +222,22 @@ def apply_manual_codes(cls, user, data, coda_input_dir):
 
         # Not everyone will have answered all of the demographic flows.
         # Label demographic questions which had no responses as TRUE_MISSING.
+        # Label data which is just the empty string as NOT_CODED.
```

Comment:
I'd really like a log as to how often this is occurring. I have a feeling that it's a problem somewhere in the upstream infrastructure rather than a surprising user behaviour

Reply:
https://github.com/AfricasVoices/Project-ADSS/pull/41